testgrid/config/config.yaml

```diff
@@ -1099,6 +1099,14 @@ dashboards:
     test_group_name: kubernetes-e2e-gce-gci-qa-slow-m57
   - name: qa-slow-master
     test_group_name: kubernetes-e2e-gce-gci-qa-slow-master
+# END google-gci-dev
+
+- name: google-gci
```

Comment:
This needs similar explanation too. May be mention that any GCI/COS specific tests that are not part of COS release qualification can be added here?